docs/aws/audit/iamcompliance/rules/root_account_access_keys.mdx

```diff
@@ -18,6 +18,29 @@ Security
 
 CBP, PCIDSS, SOC2, CIS, ISO27001, HIPAA
 
+### Remediation
+
+How to remove any active access keys created for your AWS root account?
+
+#### Using AWS Console
+
+1. Log in to the AWS Management Console using your root account credentials.
+2. Navigate to the IAM service by searching for "IAM" in the AWS services search bar and selecting "IAM" from the results.
+3. In the IAM dashboard, click on "Users" in the left sidebar.
+4. Locate the root account user and click on its username.
+5. In the user details page, navigate to the "Security credentials" tab.
+6. Under the "Access keys" section, you will see the access key associated with the root account. There may be one or multiple access keys.
+7. Click on the "Delete" button next to each access key to remove them. If you want to keep one access key for now, make sure to delete any unused or unnecessary keys.
+8. A confirmation dialog will appear. Read the message carefully and ensure that you have an alternate IAM user with the necessary permissions for management tasks. Click on the "Delete" button to proceed.
+9. Once the access key(s) are deleted, the root account will no longer have access keys associated with it.
+10. It is highly recommended to create IAM users with appropriate permissions and use those users instead of the root account for day-to-day operations. If you haven't already done so, create an IAM user and assign the necessary permissions for your regular tasks.
```

Comment:
Step 10 and onwards are not related to the rule itself. but, it's highlighting how we can improve the posture further. So, it's okay. But, otherwise, we should keep them to minimum. We would not want the readers to get confused that there is more work than what the rule is all about.